src/compare.py

```diff
@@ -1,12 +1,24 @@
 from math import sqrt
 
 import cv2
+import Levenshtein
 import numpy as np
+
+# TODO: easyocr vs. pytesseract?
+# tesseract seems to cause less overall CPU load
+# from easyocr import Reader
+# reader = Reader(["en"], gpu=False, verbose=False, download_enabled=False)
+import pytesseract
 from cv2.typing import MatLike
+from PIL import Image
 from scipy import fft
 
 from utils import BGRA_CHANNEL_COUNT, MAXBYTE, ColorChannel, ImageShape, is_valid_image
 
+# TODO: make me configureable
+pytesseract.pytesseract.tesseract_cmd = r"C:\Program Files\Tesseract-OCR\tesseract"
```

Comment:
Could we use an environment variable / `PATH` location instead?
```suggestion
pytesseract.pytesseract.tesseract_cmd = "tesseract"  # Assuming that's not already the default
```